syntax = "proto3";

import "google/protobuf/any.proto";

message SessionEvent {
	google.protobuf.Any event = 1;
}

message EventPing {}

message EventServerHandshake {
    string version = 1;
    sint32 database_version = 2;
    sint32 client_id = 3;
    sint32 image_links_version = 4;
}

message EventDatabase {
    bytes database = 1;
}

message EventImageLinks {
    bytes image_links = 1;
}
